apps/frontend/public/static/export/template.html

```diff
@@ -33,21 +33,21 @@
       <a href="#" class="px-3 py-2 text-lg font-medium">Heimdall Report</a>
       <!-- Profile info link -->
       <button role="button" aria-label="Jump to profile info card">
-        <a href="#profileInfo" class="px-3 py-2 text-sm" aria-current="page"
-          >Profile Info</a
-        >
+        <a href="#profileInfo" class="px-3 py-2 text-sm" aria-current="page">
+          Profile Info
+        </a>
       </button>
-      <!-- Control sets dropdown menu -->
-      {{#showControlSets}}
+      <!-- Result sets dropdown menu -->
+      {{#showResultSets}}
       <div class="inline-block" data-te-dropdown-ref>
         <!-- Button def -->
         <button
           type="button"
           class="inline-flex px-3 py-2 text-sm"
           data-te-dropdown-toggle-ref
-          href="#controlSetsList"
+          href="#resultSetsList"
           role="button"
-          aria-controls="controlSetsList"
```

Comment:
search replace issue here - i think it needs to stay 'aria-controls'

Reply:
Changed as requested.